share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -132,13 +87,13 @@ function(rocm_setup_version)
         if(PARSE_VERSION MATCHES "^[0-9]+\\.[0-9]+$")
             set(PARSE_VERSION ${PARSE_VERSION}.0)
         endif()
-        if(PARSE_NO_GIT_TAG_VERSION)
-            set(PACKAGE_VERSION ${PARSE_VERSION})
-        else()
-            rocm_get_commit_count(COMMIT_COUNT PARENT ${PARSE_PARENT})
-            rocm_get_version(PACKAGE_VERSION VERSION ${PARSE_VERSION}.${COMMIT_COUNT})
-        endif()
-        rocm_set_parent(PROJECT_VERSION ${PACKAGE_VERSION})
+
+        rocm_get_patch_version(ROCM_VERSION_NUM)
+	if(ROCM_VERSION_NUM)
+            set(PARSE_VERSION ${PARSE_VERSION}.${ROCM_VERSION_NUM})
```

Comment:
The rocm version should always be in the `PROJECT_VERSION_TWEAK`. This would make it the minor or patch version depending on how it was specified, which would be inconsistent.

Reply:
Updated as discussed.

Reply:
The `ROCM_VERSION_NUM` shouldn't be appended to the version. `ROCM_VERSION_NUM` should only be applied to the tweak version.